qiime2/plugin/plugin.py

```diff
@@ -173,8 +173,8 @@ def decorator(transformer):
             # Apply the decorator as we were applied with a single function
             return decorator(_fn)
 
-    def register_semantic_type_fragments(self, *semantic_type_fragments):
-        for type_fragment in semantic_type_fragments:
+    def register_type_fragments(self, *type_fragments):
```

Comment:
I don't think I am ready to deal with renaming this public API, please revert the method name to `register_semantic_types`.